packages/components/src/components/hds/stepper/list/step.hbs

```diff
@@ -0,0 +1,32 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<li class={{this.classNames}} ...attributes id={{this._stepId}} {{this._setUpStep @didInsertNode @willDestroyNode}}>
+  <div class="hds-stepper-list__step-progress">
+    <Hds::Stepper::Step::Indicator
+      @text="{{this.stepNumber}}"
+      @status={{this.status}}
+      @isInteractive={{false}}
+      class="hds-stepper-list__step-indicator"
+    />
+  </div>
+  <div class="hds-stepper-list__step-text">
+    <Hds::Text::Body class="hds-stepper-list__step-title" @tag={{this.titleTag}} @size="200" @weight="semibold">
+      {{yield to="title"}}
+      <span class="sr-only">{{this.statusSrOnlyText}}</span>
+    </Hds::Text::Body>
+    <Hds::Text::Body
+      class="hds-stepper-list__step-description"
+      @tag="div"
+      @size="100"
+      @weight="regular"
+      @color="regular"
```

Comment:
"regular" isn't one of the color values.

https://helios.hashicorp.design/components/text#textbody